physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90

```diff
@@ -21,11 +21,11 @@ module GFS_MP_generic_post
       subroutine GFS_MP_generic_post_run(                                                                                 &
         im, levs, kdt, nrcm, nncl, ntcw, ntrac, imp_physics, imp_physics_gfdl, imp_physics_thompson, imp_physics_nssl,    &
         imp_physics_mg, imp_physics_fer_hires, cal_pre, cplflx, cplchm, cpllnd, progsigma, con_g, rhowater, rainmin, dtf, &
-        frain, rainc, rain1, rann, xlat, xlon, gt0, gq0, prsl, prsi, phii, tsfc, ice, phil, htop, refl_10cm,              & 
+        frain, rainc, rain1, rann, xlat, xlon, gt0, gq0, prsl, prsi, phii, tsfc, ice, phil, htop, refl_10cm,              &
```

Comment:
These are only whitespace changes. Is that intended? If not, please revert this change.

Reply:
@grantfirl I will revert this. Thanks.